imsim/instcat.py

```diff
@@ -359,7 +353,16 @@ def getSED(self, index):
                 raise OSError("Could not find file %s in either %s or %s"%(
                               name, self.sed_dir, self.inst_dir))
             sed = galsim.SED(full_name, wave_type='nm', flux_type='flambda')
-            sed = sed.withMagnitude(0, self._bp500)  # Normalize to mag 0
+
+            # Normalize the SED to magnorm=0 at 500 nm.
+            magnorm = 0.0
+            wl = 500.0*u.nm
+            fnu = (magnorm * u.ABmag).to_value(u.erg/u.s/u.cm**2/u.Hz)
+            flambda = fnu * (astropy.constants.c/wl**2).to_value(u.Hz/u.nm)
+            hnu = (astropy.constants.h * astropy.constants.c / wl).to_value(u.erg)
+            flux_density = flambda / hnu
```

Comment:
These are all constants, so no point in redoing this calculation every time.  Should make this flux_density a class attribute so it only needs to be computed once.

Also, I was playing around with it a bit, and I think this is the cleanest way to compute this quantity with astropy.units:
```
fnu = (0 * u.ABmag).to(u.erg/u.s/u.cm**2/u.Hz)
flux_density = fnu.to_value(u.ph/u.nm/u.s/u.cm**2, u.spectral_density(500*u.nm))
```